client/src/i18n/en.json

```diff
@@ -2939,7 +2938,7 @@
    "PAYMENT_PERIOD"            : "Payment Period Management",
    "PAYROLL"                   : "Payroll",
    "PAYROLL_REPORT"            : "Payroll",
-   "PERMISSION"                : "Users and Permissions",
+   "PERMISSION"                : "Users",
```

Comment:
Why is this no longer Users and Permissions?  Why does the key PERMISSION map to "Users"?  This is very confusing.